<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoSessionFileAssociationAddDialog</class>
 <widget class="QDialog" name="QWoSessionFileAssociationAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File types like</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(237, 51, 59);</string>
       </property>
       <property name="text">
        <string>.tar.gz, .ini, .sh, .log, .conf, .db, *</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="fileTypes">
     <property name="placeholderText">
      <string>Multiple file suffix names separated by commas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Application:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="application"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnAppFind">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../private/skins/black/dirs.png</normaloff>../private/skins/black/dirs.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start parameters:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>{file} is edit file placeholder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="parameter">
     <property name="placeholderText">
      <string>&quot;{file}&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Priority:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="priority"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Example: c:\Windows\notepad.exe &quot;{file}&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
